.github/workflows/build-main.yml

```diff
@@ -3,7 +3,7 @@ name: Build Main
 on:
   push:
     branches:
-      - main
+      - main # Should this be running?
```

Comment:
This one doesn't fail, but still I'm not sure if it should be run since it wasn't before.

Reply:
It should be run when we do a push to ```main```